src/components/organisms/news/news.scss

```
@@ -0,0 +1,41 @@
+@use "../../variables" as *;
+.news {
+  .news-body {
+    border: 1px solid $gray_desertStorm;
+    & > .each-news {
+      display: flex;
```

```suggestion
  .news-body {
    border: 1px solid $gray_desertStorm;
     .each-news {
      display: flex;
```
これでも書けます。（[子孫結合子](https://developer.mozilla.org/ja/docs/Web/CSS/Descendant_combinator)）ご存じでしたらすみません。
scss -> cssに変換する際に
```
.news-body .each-news {
```
という形に変換されます。
[子結合子](https://developer.mozilla.org/ja/docs/Web/CSS/Child_combinator)との違いは、

子結合子: 直接の親子関係になければスタイルが当たらない
子孫結合子: 孫でもひ孫でも関係なく当たる
という感じです。